src/main/scala/images/Image.scala

```diff
@@ -77,8 +73,8 @@ def rotateLeft(imageFilename: String, outputFilename: String): Unit = {
 
 /** Convert an image to grayscale
   */
-def grayscale(imageFilename: String, outputFilename: String): Unit = {
-  val image = load(imageFilename)
+def grayscale(imageFilename: BufferedImage): BufferedImage = {
+  val image = imageFilename
```

Comment:
These solutions looked very similar to mine, I wonder if there is any other way this conversion could be done fairly easily. 

One suggestion I would have is to change the input variable to "image" and that way there isn't a need to assign image = imageFilename in each function since imageFilename isn't used